doc/source/conf.py

```diff
@@ -324,64 +313,44 @@
 tractor_path_linux = '/opt/pixar/Tractor-{0}'.format( tractor_version )
 tractor_path_osx = '/Applications/pixar/Tractor-{0}'.format( tractor_version )
 
-## \Todo See if the recommonmark folks would accept a patch with this
-#  functionality.
-class GafferTransform( recommonmark.transform.AutoStructify ) :
+# Add the `myst_parser` extension, which adds a parser
+# for MarkDown files with a `.md` extension.
+extensions = [ "myst_parser" ]
 
-	def auto_admonition( self, node ) :
+class GafferTransform( docutils.transforms.Transform ) :
 
-		"""
-		Replace blockquotes beginning with "admonitionType :"
-		with the equivalent admonition.
-		"""
+	default_priority = 1
```

Comment:
Assuming `blockquotes beginning with "admonitionType :"` is still how we recognize admonitions, the old comment seems more clear than "where appropriate".  Code looks fine though.

Reply:
Squashed in to https://github.com/GafferHQ/gaffer/pull/4522/commits/f30d0143f3c252a6635d666e6e3b501ea8864e53.